onebusaway-enterprise-webapp/src/main/java/org/onebusaway/enterprise/webapp/actions/api/SearchResultFactoryImpl.java

```diff
@@ -55,9 +55,9 @@ public SearchResultFactoryImpl(SearchService searchService, TransitDataService t
   }
 
   @Override
-  public SearchResult getRouteResultForRegion(RouteBean routeBean) { 
+  public SearchResult getRouteResultForRegion(RouteBean routeBean) {
     List<String> polylines = new ArrayList<String>();
-    
```

Comment:
If at all possible, try to clean up pull requests such that white space is not significant.  Its a nit, but across a larger pull request it makes it really hard to tell what was changed.  No need to make changes to this one -- but something to consider in the future